packages/fuse/fuse.spec

```diff
@@ -0,0 +1,93 @@
+
+Summary:	File System in Userspace (FUSE) v2 utilities
+License:	GPL-1.0-or-later
+URL:		http://fuse.sf.net
+Source0:	https://github.com/libfuse/libfuse/releases/download/fuse-2.9.9/fuse-2.9.9.tar.gz
+
+Patch1: fuse2-0001-More-parentheses.patch
+# https://bugzilla.redhat.com/show_bug.cgi?id=970768
+Patch2: fuse2-0002-add-fix-for-namespace-conflict-in-fuse_kernel.h.patch
+# https://github.com/libfuse/libfuse/commit/4f8f034a8969a48f210bf00be78a67cfb6964c72
+# backported for fuse2
+Patch3: fuse2-0003-make-buffer-size-match-kernel-max-transfer-size.patch
+# https://bugzilla.redhat.com/1694552#c7
+# https://github.com/libfuse/libfuse/pull/392
+# backported for fuse2
+Patch4: fuse2-0004-Whitelist-SMB2-found-on-some-NAS-devices.patch
+# cherry-picked from upstream
+Patch5:	fuse2-0005-Whitelist-UFSD-backport-to-2.9-branch-452.patch
+# cherry-picked from upstream
+Patch6: fuse2-0006-Correct-errno-comparison-571.patch
+# cherry-picked from upstream
+# https://bugzilla.redhat.com/show_bug.cgi?id=1984776
+Patch7: fuse2-0007-util-ulockmgr_server.c-conditionally-define-closefro.patch
+
+Conflicts:	filesystem < 3
+BuildRequires:	libselinux-devel
+BuildRequires:	autoconf, automake, libtool, gettext-devel, make
+
+%description
+With FUSE it is possible to implement a fully functional filesystem in a
+userspace program. This package contains the FUSE v2 userspace tools to
+mount a FUSE filesystem.
+
+%package libs
+Summary:	File System in Userspace (FUSE) v2 libraries
+License:	LGPL-2.1-or-later
+Conflicts:	filesystem < 3
+
+%description libs
+Devel With FUSE it is possible to implement a fully functional filesystem in a
+userspace program. This package contains the FUSE v2 libraries.
+
+%prep
+%autosetup -n fuse-2.9.9 -p 1
+
+# ./makeconf.sh
+#disable device creation during build/install
+sed -i 's|mknod|echo Disabled: mknod |g' util/Makefile.in
+autoreconf -ivf
+
+%build
+# Can't pass --disable-static here, or else the utils don't build
+export MOUNT_FUSE_PATH="%{_sbindir}"
+CFLAGS="%{optflags} -D_GNU_SOURCE" %configure
+make %{?_smp_mflags}
+
+%install
+mkdir -p %{buildroot}%{_cross_bindir}
+mkdir -p %{buildroot}%{_cross_sbindir}
+mkdir -p %{buildroot}%{_cross_libdir}
+mkdir -p %{buildroot}%{_cross_licensedir}/%{_uncross_name}
+
+install -m 0755 lib/.libs/libfuse.so.%{version} %{buildroot}%{_cross_libdir}
+install -m 0755 lib/.libs/libulockmgr.so.1.0.1 %{buildroot}%{_cross_libdir}
+install -m 0755 util/fusermount %{buildroot}%{_cross_bindir}
+install -m 0755 util/mount.fuse %{buildroot}%{_cross_sbindir}
+install -m 0755 util/ulockmgr_server %{buildroot}%{_cross_bindir}
+# Get rid of static libs
+rm -f %{buildroot}%{_cross_libdir}/*.a
+
+%ldconfig_scriptlets libs
+
+ls -l %{buildroot}%{_cross_bindir}
+ls -l %{buildroot}%{_cross_sbindir}
+ls -l %{buildroot}%{_cross_libdir}
+ls -l %{buildroot}%{_cross_licensedir}
+
+%files
+%license COPYING
+%{_cross_attribution_file}
+%{_cross_sbindir}/mount.fuse
+%attr(4755,root,root) %{_cross_bindir}/fusermount
```

Comment:
You should only have `%{_cross_bindir}/fusermount`.

Reply:
Should I? The fusermount executable is installed set-uid-root. Anything that tries to fork a fusermount on bottlerocket is likely to already be root, so maybe we don't need to make the executable suid-root. What do you think?

Reply:
I think you are right, because the systemd services will be running as UID 0.